tests/modules/vpc_sc/test_plan.py

```diff
@@ -12,9 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 def test_create_policy(plan_runner):
   "Test with auto-created policy."
-  _, resources = plan_runner()
+  access_policy_create = '''{
+    parent = "organizations/123456"
+    title  = "vpcsc-policy"
+  }'''
+  _, resources = plan_runner(tf_var_file='test.regular.tfvars',
+                             access_policy='null',
+                             access_policy_create=access_policy_create)
   counts = {}
```

Comment:
`collections.Counter` (see [here](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/tests/modules/cloudsql_instance/test_plan.py#L113)). I know you'll say no but this is very python 2.2.

Reply:
oh wow, I should go and check up on collections from time to time

Reply:
done, thanks